doc/users/next_whats_new/engformatter_offset.rst

```diff
@@ -0,0 +1,14 @@
+:class:`matplotlib.ticker.EngFormatter` now computes offset by default
+----------------------------------------------------------------------
+
+:class:`matplotlib.ticker.EngFormatter` has gained the ability to show an
+offset text near the axis. With shared logic with
+:class:`matplotlib.ticker.ScalarFormatter`, it is capable of deciding whether
+the data qualifies having an offset and show it with an appropriate SI quantity
+prefix, and with the supplied ``unit``.
```

Comment:
I'm still not fluent in Rst and since I'm also not sure what were your intentions I'll ask: You wanted to remove the links? Why? Also, if you want it to simply result in `<code>` formatted text, why not use double backticks?

Reply:
The suggested text is still a reference, and is less clutter for reading the raw text; code is double backticks, and unrelated.